### PR TITLE
Hotfix/Print view

### LINKED ISSUE #223: Print View Fixes
- [ ] Fix css class for ODP 
- [ ] Fix exception on ODP

### PR DESCRIPTION
**Pull request by minotogna**
Resolves #223

### CODE REVIEW

File: webapp/app/assessment/components/dataTable/chart/components/noDataPlaceholder.js [hunk 1]
```diff
@@ -96,9 +96,9 @@ class NoDataPlaceholder extends React.Component {
     const { wrapperWidth, i18n } = this.props
 
     return <g className="chart__no-data-placeholder" ref="container">
-      <image ref="tucan" href="/img/tucan.svg" width={tucanWidth} height={tucanHeight} x={this.getTucanX()} y={tucanY}/>
-      <Text text={i18n.t('extentOfForest.chart.placeholderLine1')} width={wrapperWidth} y="144"/>
-      <Text text={i18n.t('extentOfForest.chart.placeholderLine2')} width={wrapperWidth} y="164"/>
+      <image ref="tucan" href="/img/tucan.svg" width={tucanWidth} height={tucanHeight} x={this.getTucanX()} y={tucanY} style={{opacity: 0}}/>
+      <Text text={i18n.t('extentOfForest.chart.placeholderLine1')} width={wrapperWidth} y="144" style={{opacity: 0}}/>
+      <Text text={i18n.t('extentOfForest.chart.placeholderLine2')} width={wrapperWidth} y="164" style={{opacity: 0}}/>
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/chart/components/noDataPlaceholder.js:101]: Can we use className?

File: webapp/app/assessment/components/dataTable/table/cell/select.js [hunk 2]
```diff
@@ -39,12 +41,12 @@ const Select = props => {
         className="fra-table__select no-print"
         value={datum || optionNotSelected[SectionSpec.KEYS_COL.optionName]}
         disabled={disabled}
-        onChange={e => {
+        onChange={(e) => {
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/cell/select.js:44]: e => event

File: webapp/app/assessment/components/dataTable/table/rowData.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ const RowData = props => {
       </th>
 
       {odp
-        ? data.map(datum => (
+        ? data.map((datum) => (
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/rowData.js:49]: Can it be mapped directly to CellOdp
`data.map(CellOdp)`

File: webapp/app/assessment/components/dataTable/table/rowData.js [hunk 4]
```diff
@@ -60,7 +60,7 @@ const RowData = props => {
               updateTableDataCell={updateTableDataCell}
             />
           ))
-        : colsData.map(col => (
+        : colsData.map((col) => (
```
Comment (sorja) [inline webapp/app/assessment/components/dataTable/table/rowData.js:63]: Can it be mapped directly to Cell
`colsData.map(Cell)`

File: webapp/app/assessment/components/dataTable/chart/components/noDataPlaceholder.js (hunk 1 above)
Comment (minotogna) [reply webapp/app/assessment/components/dataTable/chart/components/noDataPlaceholder.js:101]: This is just a quick fix... the all chart components must be refactored.. let's do this later on if ok for you as well

File: webapp/app/assessment/components/dataTable/table/cell/select.js (hunk 2 above)
Comment (minotogna) [reply webapp/app/assessment/components/dataTable/table/cell/select.js]: yep, committed

**Pull request merged**